static/responses/json/3/action/package_search?sort=id+asc&start=0&rows=100

```diff
@@ -147,10 +147,6 @@
         "url": "<!--#echo var="mock_absolute_root_url" -->mock-third-party/example-dataset-1/about",
         "notes": "This is an example CKAN dataset consisting of a number of active resources.",
         "owner_org": "9f3af52d-7a2e-4962-95c7-15a23b6e0b82",
-        "author": "Nobody",
-        "author_email": "nobody@example.com",
-        "maintainer": "Nobody",
-        "maintainer_email": "nobody@example.com",
```

Comment:
I specifically added these fields so that we could be sure CKAN discarded them.

Reply:
OK, will add back in